aspnetcore/tutorials/getting-started-with-swashbuckle/includes/getting-started-with-swashbuckle8.md

```diff
@@ -2,7 +2,12 @@
 :::moniker range=">= aspnetcore-6.0 < aspnetcore-9.0"
 
 > [!NOTE]
-> Swashbuckle is not available in .NET 9 or later. For an alternative, see <xref:fundamentals/openapi/overview?view=aspnetcore-9.0&preserve-view=true>.
+> In .NET 9 and later, ASP.NET Core includes built-in OpenAPI support. Swashbuckle is no longer included by default, but it remains available as a community package you can add manually to ASP.NET Core projects targeting .NET 9 or later.
+> 
+> • To understand the built‑in OpenAPI features, see <xref:fundamentals/openapi/overview?view=aspnetcore-9.0&preserve-view=true>.  
+> • To add and use the Swagger UI provided by the `Swashbuckle.AspNetCore.SwaggerUI` package for interactive exploration or local ad‑hoc testing, see <xref:fundamentals/openapi/using-openapi-documents#use-swagger-ui-for-local-ad-hoc-testing>.
+>
+>The following instructions apply when using Swashbuckle with .NET versions earlier than 9.
 
 There are three main components to Swashbuckle:
```

Comment:
The swashbuckle version (6.6.2) of the instructions is outdated (5/21/2024).
It could be updated to the latest version (9.0.6), which should not impact the code.
If you want I can upgrade the .NET 9/10 examples and update the version in the docs as well.

Reply:
Thanks @timdeschryver, sorry for the delay, I just saw this.

It is intentional to not carry this tutorial forward beyond the old 8 version and what was compatibale with that version at the time.  Eventually this one will be retiured.